.gitpod.yml

```diff
@@ -4,7 +4,19 @@ image:
 checkoutLocation: dazzle
 workspaceLocation: dazzle
 tasks:
+  - command: sudo buildkitd --group gitpod
+  - command: docker run -p 5000:5000 registry
```

Comment:
Use registry:2 tag instead of latest to be more explicit and less - albeit minimal - likelihood of related issues?